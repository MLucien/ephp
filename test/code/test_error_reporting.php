<?php

error_reporting(E_ALL);

$b = $a;

error_reporting(E_ERROR);

$c = $d;
