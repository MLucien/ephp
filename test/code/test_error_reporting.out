
Notice: Undefined variable: a in {{CWD}}/test/code/test_error_reporting.php on line 5
